packages/firestore/test/integration/api/aggregation.test.ts

```diff
@@ -18,24 +18,125 @@
 import { expect } from 'chai';
 
 import {
+  collection,
+  collectionGroup,
   countQuery,
+  doc,
   getAggregateFromServerDirect,
-  query
+  query,
+  terminate,
+  where,
+  writeBatch
 } from '../util/firebase_export';
-import { apiDescribe, withTestCollection } from '../util/helpers';
+import {
+  apiDescribe,
+  postConverter,
+  withTestCollection,
+  withTestDb
+} from '../util/helpers';
 
 apiDescribe('Aggregation query', (persistence: boolean) => {
   it('can run count query getAggregateFromServerDirect', () => {
     const testDocs = {
-      a: { k: 'a', sort: 1 },
-      b: { k: 'b', sort: 2 },
-      c: { k: 'c', sort: 2 }
+      a: { author: 'authorA', title: 'titleA' },
+      b: { author: 'authorB', title: 'titleB' }
+    };
+    return withTestCollection(persistence, testDocs, async collection => {
+      const countQuery_ = countQuery(collection);
+      const snapshot = await getAggregateFromServerDirect(countQuery_);
+      expect(snapshot.getCount()).to.equal(2);
+    });
+  });
+
+  it('aggregateQuery.query equals to original query', () => {
+    const testDocs = {
```

Comment:
This is also a case where the actual contents of the collection are not relevant to the test case. For test cases where the contents of the collection don't matter, please just use an empty collection. By using a non-empty collection it distracts the reader from the purpose of the test case. I believe there is even a withEmptyTestCollection() helper method you could use. Please make the same change in other test methods below.

Reply:
Lite-api test helpers has withTestCollection, which is pointed to an empty collection, but api test helpers doesn't have one similar. So created a simple withEmptyTestCollection helper in this PR.